src/store/middleware/check-auth.js

```diff
@@ -70,8 +70,7 @@ export default ({ getState }) =>
         checkLoggedIn(state, wsControllerURL)
       ) {
         // Await the next to support async thunks
-        await next(action);
-        return;
+        return await next(action);
```

Comment:
I was stumped for ages about why the promises were getting lost in the middleware and it turned out that this middleware was not returning the next promise.

Reply:
Wow I can imagine how frustrating this is!